x-pack/plugins/security_solution/server/fleet_integration/handlers/create_default_policy.ts

```diff
@@ -50,9 +55,15 @@ const getEndpointPolicyWithIntegrationConfig = (
   policy: PolicyConfig,
   config: PolicyCreateEndpointConfig | undefined
 ): PolicyConfig => {
+  const isNGAV = config?.endpointConfig?.preset === ENDPOINT_CONFIG_PRESET_NGAV;
   const isEDREssential = config?.endpointConfig?.preset === ENDPOINT_CONFIG_PRESET_EDR_ESSENTIAL;
+  const isEDRComplete = config?.endpointConfig?.preset === ENDPOINT_CONFIG_PRESET_EDR_COMPLETE;
+
+  const isOnlyEvents = !isNGAV && !isEDREssential && !isEDRComplete;
```

Comment:
hey @gergoabraham

What about adding a preset for data collection, and making it the default preset?

For example:

```js

/**
 * Retrieve endpoint config preset defaulting to data collection
 */
const getEndpointConfigPreset = (config: PolicyCreateEndpointConfig | undefined) => {
  if (config?.endpointConfig?.preset) {
    return config.endpointConfig.preset;
  }

  return ENDPOINT_CONFIG_PRESET_DATA_COLLECTION;
};

/**
 * Boolean helper for endpointConfigPresets
 */
const endpointConfigPresetHelper = (config: PolicyCreateEndpointConfig | undefined) => {
  const preset = getEndpointConfigPreset(config);

  return {
    isNGAV: preset === ENDPOINT_CONFIG_PRESET_NGAV,
    isEDREssential: preset === ENDPOINT_CONFIG_PRESET_EDR_ESSENTIAL,
    isEDRComplete: preset === ENDPOINT_CONFIG_PRESET_EDR_COMPLETE,
    isDataCollection: preset === ENDPOINT_CONFIG_PRESET_DATA_COLLECTION,
  };
};


/**
 * Retrieve policy for endpoint based on the preset selected in the endpoint integration config
 */
const getEndpointPolicyWithIntegrationConfig = (
  policy: PolicyConfig,
  config: PolicyCreateEndpointConfig | undefined
): PolicyConfig => {
  const { isNGAV, isEDREssential, isDataCollection } = endpointConfigPresetHelper(config);

  if (isDataCollection) {
		return disableProtections(policy);
  } else if (isNvGAV || isEDREssential) {
	// ... then everything is the same
```

I believe that would speed up the work for this ticket: https://github.com/elastic/security-team/issues/5063